### PR TITLE
Accessibility: adding a disabled text to aria label

### PR DESCRIPTION
**Pull request by JoshuaLai**
# What
Defect raised by accessibility team, participant items that cannot be interacted with should say "disabled"
<div> and <stack> did not have a `disabled` prop, so we are using the aria label and appending "disabled" after the participants name if there is no menu popover

# Why
Accessibility defect 

# How Tested
Start 2 Chat Sample 
Have 2 participants in the chat 
Open participant pane
Turn on voice over
Cycle through the participants. 

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [x] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

**Review by edwardlee-msft — APPROVED** [review]

**Comment by edwardlee-msft** [comment]
Looks great! Thanks for applying this fix. It definitely also applies to calling as well so you helped us out by getting this in :)

**Review by palatter — APPROVED** [review]

File: packages/react-components/src/components/ParticipantItem.tsx [hunk 1]
```diff
@@ -269,6 +274,11 @@ export const ParticipantItem = (props: ParticipantItemProps): JSX.Element => {
         })}
         id={uniqueId}
         aria-labelledby={`${props.ariaLabelledBy} ${uniqueId}`}
+        aria-label={
+          menuItems && menuItems.length > 0
+            ? ariaDisplayName
+            : strings.disabled.replace('{user}', ariaDisplayName || '')
```
Comment (JamesBurnside) [inline packages/react-components/src/components/ParticipantItem.tsx:280]: Is using [aria-disabled](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-disabled) not sufficient?

Comment (edwardlee-msft) [reply packages/react-components/src/components/ParticipantItem.tsx:280]: did a little bit of digging and yes I htink aria-disabled=true would work here as well!
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-disabled

@JoshuaLai , my bad! If this works, then this is probably the better approach here

Comment (JoshuaLai) [reply packages/react-components/src/components/ParticipantItem.tsx]: Fixed up thanks for the call out. Using aria-disabled on the menu-item div works. Also as a note on Windows voice over says "Disabled" on MacOS voice over says "Dimmed"

**Comment by JoshuaLai** [comment]
Checked the snapshots and behaviour nothing has really changed so we can safely update this.

**Pull request merged**